backend/analysis-manager/main.py

```diff
@@ -10,8 +20,20 @@ def handler(request: Request) -> typing.ResponseReturnValue:
     :param request: the incoming request
     :return: an empty successful response
     """
-
     # TODO: validate incoming request
+    try:
+        validate_request = AnalyzeTextAreaModel(
+            text_body=str(request.form.get("text-to-analyze")),
+            author_name=str(request.form.get("author")).split(","),
+            publisher=str(request.form.get("publisher")),
+            source_url=str(request.form.get("source-url")),
+        )
+    except ValidationError as e:
+        response = make_response({error["type"]: error["msg"] for error in e.errors()})
```

Comment:
From what I remember testing this in the cache handler, you don't need the `make_reponse` call.

Additionally, we should return a non-200 status code. This can be done by returning a tuple of the response body followed by the status code (i.e. `return {"hello": "world"}, 400`).

Reply:
Good catch on the status code. How about 422?

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422

I personally disagree on returning in that way even if it's shorter.  I think using the Flask response object makes sense even if it adds a couple of lines. If we decide to add other things to the response later I feel this will be more clear.

```
response = make_response("body")
response.status_code(400)
return response

```

If you still disagree I'll change it, but maybe let's consider if there's any real disadvantage of doing it this way first.

Reply:
`422` is probably the right one. I think it's also what FastAPI uses when validating input, so we'd be in good company if nothing else.

That's fair. The response return style is much more of a personal preference.